setup.py

```diff
@@ -114,7 +114,7 @@ def call_package_config(package, option, optional=False):
 ]
 
 libraries = ['stdc++']
-extra_compile_args = ['-Wall', '-O0'] + png_flags
+extra_compile_args = ['-Wall', '-O0', '-std=c++17'] + png_flags
```

Comment:
just noticed this. We definitely don't want -O0 in most cases.

Reply:
I think that's actually shadowed by distutils when creating the final compiler argument list. Still, I agree we don't want that there.